src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -131,42 +97,6 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
             }
         }
 
-
-        [Test]
-        public void Setter_dependencies_should_be_supported()
```

Comment:
Hmm, not sure I agree, while we can state that we internally won't use Property Injection and our default builder won't support it I'd say its more of a stretch to say what the external builders won't support it. Esp since all  of them supports it now?